packages/flutter_test/lib/src/matchers.dart

```diff
@@ -1646,28 +1646,23 @@ class _MatchesReferenceImage extends AsyncMatcher {
     final TestWidgetsFlutterBinding binding = TestWidgetsFlutterBinding.ensureInitialized();
     return binding.runAsync<String>(() async {
       final ui.Image image = await imageFuture;
-      final ByteData bytes = await image.toByteData()
-        .timeout(const Duration(seconds: 10), onTimeout: () => null);
-      if (bytes == null) {
-        return 'Failed to generate an image from engine within the 10,000ms timeout.';
-      }
+      final ByteData bytes = await image.toByteData();
+      if (bytes == null)
+        return 'could not be encoded.';
```

Comment:
Nit: "Image could not be encoded"

Reply:
Oh n/m - this is going to be appended to the matcher's failure reason, preceded by a "which" isn't it?

Reply:
I'm guessing yes? The code was inconsistent before, I made it all use this style.